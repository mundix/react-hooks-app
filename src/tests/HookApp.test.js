import React from 'react';
import { shallow } from 'enzyme';
import { HookApp } from '../HookApp';


describe('PRueba en <HookApp/>', () => {
    
    test('debe mostrarse correctamente', ()=>{
        const wrapper = shallow( <HookApp/> );
        expect( wrapper).toMatchSnapshot();
    });

});