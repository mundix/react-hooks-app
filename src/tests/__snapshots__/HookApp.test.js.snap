// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PRueba en <HookApp/> debe mostrarse correctamente 1`] = `
<div>
  Hola Mundo
</div>
`;
